index.html

```diff
@@ -0,0 +1,289 @@
+                            If you like what you see and have a project you need coded,
+                            don't hesitate to contact me.
+                        </p>
+                    </div>
+                    <nav id="lets-connect">
+                        <h5>LET'S CONNECT</h5>
+                    </nav>
+                    <div>
+                        <ul class="icons">
+                            <li><img src="Vector.png" alt=""></li>
+                            <li><img src="Linkedin.png" alt=""></li>
+                            <li><img src="Vector1.png" alt=""></li>
+                            <li><img src="Vector3.png" alt=""></li>
+                        </ul>
+                    </div>
+                    <section class="bottom-images">
+                        <div id="elipse12">
+                            <img src="Ellipse12.png" alt="">
+                        </div>
+                        <div id="elipse13">
+                            <img src="Ellipse13.png" alt="">
+                        </div>
+                        <div id="elipse15">
+                            <img src="Ellipse15.png" alt="">
+                        </div>
+                        <div id="elipse4">
+                            <img src="Ellipse4.png" alt="">
+                        </div>
+                        <div id="rectangle5">
+                            <img src="Rectangle5.png" alt="">
+                        </div>
+                    </section>
+                </section>
+            </div>
+    </article>
+        
+        <div class="work-card">
+            <div class="work1">
+                <img src="Snapshoot.png" alt="">
+               <div id="block1">
+                <h2>Tonic</h2>
+                <div id="frame">
+                    <ul>
+                        <li><div id="text1"><h5>CANOPY</h5></div></li>
+                        <li><div id="counter"></div></li>
+                        <li><div id="role"><h5>Back End Dev</h5></div></li>
+                        <li><div id="counter1"></div></li>
+                        <li><div id="text"><h5>2015</h5></div></li>
+                    </ul>
+                </div>
+                <div class="primary-text">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="tags">
+                    <ul>
+                        <li><div id="tag1"><div id="html1">html</div></div></li>
+                        <li><div id="tag2"><div id="css1">css</div></div></li>
+                        <li><div id="tag3"><div id="javascript">javascript</div></div></li>
+                    </ul>
+                        
+                </div>
+                <div class="Actions">
+                    <button id="see-project">
+                        <b>See Project</b>
+                    </button>
```

Comment:
- [ ] According to the requirements, You should Implement the button interactions (enable, hover, etc.). Kindly, add the button interactions based on your Figma Design 👌.

![1](https://user-images.githubusercontent.com/45231276/176833382-1d1d826d-6c66-47ac-bec8-36427809918f.PNG)

![image](https://user-images.githubusercontent.com/45231276/177019437-d4c20bc0-7e61-4d62-ac96-2b0d099054c1.png)

- [ ] Note: you should do this on all buttons 👌